.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
```

Comment:
Which step needs this file? Where do we use it?

Reply:
:+1: 
We were using it before when I was using Dockerfile and docker instead of JIB
Removed